views/partials/form.handlebars

```diff
@@ -1 +1,49 @@
-Edit topic form
\ No newline at end of file
+<!-- Edit topic form -->
+<form>
+    <div class="row">
+            <div class="col-lg-4">
+                <div class="form-group">
+                    <input type="text" placeholder="subjet name" class="form-control">
+                </div>
+            </div>
+            <div class="col-lg-1">
+                <div class="form-group">
+                    <input type="number"  class="form-control">
+                </div>
+            </div>
+            <div class="col-lg-1">
+                <div class="form-group">
+                    <input type="number"  class="form-control">
+                </div>
+            </div>
+            <div class="col-lg-1">
+                <div class="form-group">
+                    <input type="number"  class="form-control">
+                </div>
+            </div>
+            <div class="col-lg-1">
+                <div class="form-group">
+                    <input type="number"  class="form-control">
+                </div>
+            </div>
+            <div class="col-lg-1">
+                <div class="form-group">
+                    <input type="number"  class="form-control">
+                </div>
+            </div>
+            <div class="col-lg-1">
+                <div class="form-group">
+                    <input type="number"  class="form-control">
+                </div>
+            </div>
+            <div class="col-lg-1">
+                <div class="form-group">
+                    <input type="number"  class="form-control">
+                </div>
+            </div>
+            <div class="col-lg-1">
+                <button type="button">Edit</button>
```

Comment:
the form won't submit if we have `type="button"` on here, once clicked

could we please remove it? and can we please change it's name? This will be used on both the add page and the edit page, so we need a generic name
`<button>Submit</button>`